tests/parsers/test_tflint.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+
+import codecs
+import os
+
+import inlineplz.parsers.tflint as tflint
+
+
+tflint_path = os.path.join(
+    "tests", "testdata", "parsers", "tflint.01-testcase.tf.stdout"
+)
+
+
+def test_tflint():
+    with codecs.open(tflint_path, encoding="utf-8", errors="replace") as inputfile:
```

Comment:
I'm using `backslashreplace` instead of `replace`, we should pick one and standardize around it for consistency